libioc/Config/Jail/Defaults.py

```diff
@@ -88,22 +103,38 @@
     "exec_prestop": None,
     "exec_stop": "/bin/sh /etc/rc.shutdown",
     "exec_poststop": None,
+    "exec_system_user": "root",
+    "exec_system_jail_user": "root",
```

Comment:
What is the difference to the `exec_jail_user` config property, that already exists?

Reply:
Hi @gronke.
I'm sorry.  This parameter is property via `iocage` config(First, After My case did FreeBSD host machine installed iocage, jail did created or deleted via libioc).Therefore, FreeBSD host machine had return but iocage response, not libioc response.
Therefore, this parameter doesn't need config it(Maybe Other too).
I remove config it.

Reply:
Now, My ZFS filesystem had recreate `iocage` directory from `iocage` tool to `libioc` tool.

Therefore, it isn't said `unknown config propetery`.

Reply:
The config property indeed exists in python-iocage, see:

https://github.com/iocage/iocage/blob/528f007e92f6e081614d8a09186c4831fc9f3e32/iocage.8#L1299-L1314

| Name | Default | Description |
| ------ | ------- | ----------- |
| exec_system_jail_user | `0` | This boolean option looks for the exec_jail_user file in the system rather than the jail's file. | 
| exec_system_user | `root` | Run commands as this user in the system environment. The default is to run commands as the current user. |

We can track the missing feature in #662.